src/nvim/window.c

```diff
@@ -3028,10 +3048,10 @@ int make_tabpages(int maxcount)
   return count - todo;
 }
 
-/*
- * Return TRUE when "tpc" points to a valid tab page.
- */
-bool valid_tabpage(tabpage_T *tpc)
+/// Return true when "tpc" points to a valid tab page.
+///
+/// @param tpc tabpage to check
```

Comment:
```C
/// Check that tpc points to a valid tab page
///
/// @param[in]  tpc  Tabpage to check.
```

(`@return` section contents is obvious, so only summary which does not look good if it looks like `@return` section.)